### PR TITLE
Trust nexus server certificate, as a input parameter to the task

### PR DESCRIPTION
**Pull request by ypupo2002**

### CODE REVIEW

File: Extension/artifactUpload/task.json [hunk 1]
```diff
@@ -89,6 +89,14 @@
...
         "required": false,
         "helpMarkDown": "The file extension of the file to upload. If empty, the default file extension (derived from the packaging type) will be used."
+      },
+      {
+        "name": "trustSSL",
+        "type": "boolean",
+        "label": "Trust Server Certificate",
+        "defaultValue": "false",
+        "required": false,
+        "helpMarkDown": "Trust Nexus Server certificate (usefull in case of self signed certificate present on Nexus)"
```
Comment (davidstaheli) [inline Extension/artifactUpload/task.json]: Can you please change this `helpMarkDown` to the following?

"Selecting this option results in the Nexus Server's SSL certificate being trusted even if it is self-signed or cannot be validated by a Certificate Authority (CA)."

**Comment by davidstaheli** [comment]
Thanks Yunior.  I'm sorry but I just noticed one other change to be made.  The patch number of the version needs to be incremented so that people automatically get the update.  Can you please increment the version from **0.1.2** to **0.1.3** in the files below?  Then I think we can merge this pull request.  Thank you.

Files:
Extension/**extension-manifest.json**
Extension/artifactUpload/**package.json**
Extension/artifactUpload/**task.json**
Extension/artifactUpload/**task.loc.json**

**Comment by ypupo2002** [comment]
OK, I will update the branch with the version bump in a couple of minutes.

**Comment by davidstaheli** [comment]
Thanks Yunior!

**Pull request merged**